Apps/Database/src/Models/Comment.cs

```diff
@@ -0,0 +1,64 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Database.Models
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    /// <summary>
+    /// A user entered Comment.
+    /// </summary>
+    public class Comment : AuditableEntity
+    {
+        /// <summary>
+        /// Gets or sets the id.
+        /// </summary>
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Column("CommentId")]
+        public Guid Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets the user hdid.
+        /// </summary>
+        [Column("UserProfileId")]
```

Comment:
I think we should be creating relationships in our models for the DB or at the very least in the DB Context.  Doing this will automatically link the tables but does require that we use the proper column name:

        [MaxLength(52)]
        public string? UserProfileId { get; set; }

        public virtual UserProfile? UserProfile { get; set; }

I think that going forward we shouldn't use HDID in our code as it is an external ID and instead it should be locally referenced as UserProfileID.